.gitpod.yml

```diff
@@ -0,0 +1,17 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
```

Comment:
This file probably belongs to another PR

Reply:
Yes, this was used to configure Gitpod so that I can preview. How can I move it to another PR? Do I have to delete this first and then create it again?